distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-api-contents.adoc

```diff
@@ -0,0 +1,247 @@
+
+==== Catalog API
+
+The Catalog API is an OSGi bundle (`catalog-core-api`) that contains the Java interfaces for the Catalog components and implementation classes for the Catalog Framework, Operations, and Data components.
+
+===== Catalog API Search Interfaces
+
+====== ${ddf-ui} Application Search Interface
+
+The ${branding} Search UI application provides a graphic interface to return results and locate them on an interactive globe or map.
+
+====== SSH Search Interface
+
+Additionally, it is possible to use a client script to remotely access ${branding} via SSH and send console commands to search and ingest data.
+
+===== Catalog Search Result Objects
+
+Data is returned from searches as Catalog Search Result objects.
```

Comment:
We should work this section a bit. Instead of Catalog Entry here we are writing about `Metacards` and `Results`. http://codice.org/ddf/apidocs/index.html?ddf/catalog/data/Result.html